lib/template/places-json.php

```diff
@@ -0,0 +1,45 @@
+<?php
+	$aFilteredPlaces = array();
+
+	if (!sizeof($aPlaces))
+	{
+		if (isset($sError))
+			$aFilteredPlaces['error'] = $sError;
+		else
+			$aFilteredPlaces['error'] = 'Unable to geocode';
```

Comment:
I basically used https://github.com/twain47/Nominatim/blob/master/lib/template/address-jsonv2.php as base for my new file. This part was copied from there.